test/Riok.Mapperly.IntegrationTests/Mapper/StaticTestMapper.cs

```diff
@@ -16,8 +16,6 @@ public static partial class StaticTestMapper
 
         public static partial long ImplicitCastInt(int value);
 
-        public static partial int ExplicitCastInt(uint value);
```

Comment:
Here, the explicit cast was tested explicitly. Instead of removing it, I'd enable the explicit cast (using `.All`) for this mapper.